pango/src/attribute.rs

```diff
@@ -14,96 +20,153 @@ use crate::Weight;
         unsafe { from_glib_full(ffi::pango_attr_background_new(red, green, blue)) }
     }
 
+    #[doc(alias = "pango_attr_fallback_new")]
     pub fn new_fallback(enable_fallback: bool) -> Option<Attribute> {
         unsafe { from_glib_full(ffi::pango_attr_fallback_new(enable_fallback.to_glib())) }
     }
 
+    #[doc(alias = "pango_attr_family_new")]
     pub fn new_family(family: &str) -> Option<Attribute> {
         unsafe { from_glib_full(ffi::pango_attr_family_new(family.to_glib_none().0)) }
     }
 
     #[cfg(any(feature = "v1_38", feature = "dox"))]
     #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_38")))]
+    #[doc(alias = "pango_attr_font_features_new")]
     pub fn new_font_features(features: &str) -> Option<Attribute> {
         unsafe { from_glib_full(ffi::pango_attr_font_features_new(features.to_glib_none().0)) }
     }
 
     #[cfg(any(feature = "v1_38", feature = "dox"))]
     #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_38")))]
+    #[doc(alias = "pango_attr_foreground_alpha_new")]
     pub fn new_foreground_alpha(alpha: u16) -> Option<Attribute> {
         unsafe { from_glib_full(ffi::pango_attr_foreground_alpha_new(alpha)) }
     }
 
+    #[doc(alias = "pango_attr_foreground_new")]
     pub fn new_foreground(red: u16, green: u16, blue: u16) -> Option<Attribute> {
         unsafe { from_glib_full(ffi::pango_attr_foreground_new(red, green, blue)) }
     }
 
+    #[doc(alias = "pango_attr_gravity_hint_new")]
     pub fn new_gravity_hint(hint: GravityHint) -> Option<Attribute> {
         unsafe { from_glib_full(ffi::pango_attr_gravity_hint_new(hint.to_glib())) }
     }
 
+    #[doc(alias = "pango_attr_gravity_new")]
     pub fn new_gravity(gravity: Gravity) -> Option<Attribute> {
         unsafe { from_glib_full(ffi::pango_attr_gravity_new(gravity.to_glib())) }
     }
 
+    #[cfg(any(feature = "v1_44", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_44")))]
+    #[doc(alias = "pango_attr_insert_hyphens_new")]
+    pub fn new_insert_hyphens(insert_hyphens: bool) -> Option<Attribute> {
+        unsafe { from_glib_full(ffi::pango_attr_insert_hyphens_new(insert_hyphens.to_glib())) }
+    }
+
+    #[doc(alias = "pango_attr_letter_spacing_new")]
     pub fn new_letter_spacing(letter_spacing: i32) -> Option<Attribute> {
         unsafe { from_glib_full(ffi::pango_attr_letter_spacing_new(letter_spacing)) }
     }
 
+    #[cfg(any(feature = "v1_46", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_46")))]
+    #[doc(alias = "pango_attr_overline_color_new")]
+    pub fn new_overline_color(red: u16, green: u16, blue: u16) -> Option<Attribute> {
+        unsafe { from_glib_full(ffi::pango_attr_overline_color_new(red, green, blue)) }
+    }
+
+    #[cfg(any(feature = "v1_46", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_46")))]
+    #[doc(alias = "pango_attr_overline_new")]
+    pub fn new_overline(overline: Overline) -> Option<Attribute> {
```

Comment:
Are all these constructors actually `nullable`? That seems strange for the majority of them

Reply:
I would guess that none of them are nullable. I will remove Option from all of them.